resources/js/views/settings/servers/View.vue

```diff
@@ -0,0 +1,337 @@
+          </b-button>
+        </div>
+      </template>
+
+      <b-form @submit='saveServer' :aria-hidden="modelLoadingError">
+        <b-container fluid>
+          <b-form-group
+            label-cols-sm='4'
+            :label="$t('settings.servers.description')"
+            label-for='description'
+            :state='fieldState("description")'
+          >
+            <b-form-input id='description' type='text' v-model='model.description' :state='fieldState("description")' :disabled='isBusy || modelLoadingError || viewOnly'></b-form-input>
+            <template slot='invalid-feedback'><div v-html="fieldError('description')"></div></template>
+          </b-form-group>
+          <b-form-group
+            label-cols-sm='4'
+            :label="$t('settings.servers.base_url')"
+            label-for='base_url'
+            :state='fieldState("base_url")'
+          >
+            <b-form-input id='base_url' type='text' v-model='model.base_url' :state='fieldState("base_url")' :disabled='isBusy || modelLoadingError || viewOnly'></b-form-input>
+            <template slot='invalid-feedback'><div v-html="fieldError('base_url')"></div></template>
+          </b-form-group>
+          <b-form-group
+            label-cols-sm='4'
+            :label="$t('settings.servers.salt')"
+            label-for='salt'
+            :state='fieldState("salt")'
+          >
+            <b-input-group>
+              <b-form-input id='salt' :type='hideSalt ? "password": "text"' v-model='model.salt' :state='fieldState("description")' :disabled='isBusy || modelLoadingError || viewOnly'></b-form-input>
+              <b-input-group-append>
+                <b-button @click="hideSalt = !hideSalt" variant="success"><b-icon-eye-fill v-if="hideSalt"></b-icon-eye-fill><b-icon-eye-slash-fill v-else></b-icon-eye-slash-fill></b-button>
+              </b-input-group-append>
+            </b-input-group>
+            <template slot='invalid-feedback'><div v-html="fieldError('salt')"></div></template>
+          </b-form-group>
+          <b-form-group
+            :description="$t('settings.servers.strength_description')"
+            label-cols-sm='4'
+            :label="$t('settings.servers.strength')"
+            label-for='strength'
+            :state='fieldState("strength")'
+          >
+            <b-form-rating id='strength' stars="10" v-model='model.strength' :state='fieldState("strength")' :disabled='isBusy || modelLoadingError || viewOnly'>
+            </b-form-rating>
+            <template slot='invalid-feedback'><div v-html="fieldError('strength')"></div></template>
+          </b-form-group>
+
+          <b-form-group
+            label-cols-sm='4'
+            :label="$t('settings.servers.disabled')"
+            label-for='disabled'
+            :state='fieldState("disabled")'
+          >
+            <b-form-checkbox id="disabled" v-model="model.disabled" name="check-button" switch :disabled='isBusy || modelLoadingError || viewOnly'>
```

Comment:
The checkbox stays higher than the label, since it gets not centered. A solution can be found here: https://github.com/THM-Health/PILOS/blob/e6bdd60f9c473ecd4e81b5807f568bb2bbf18669/resources/js/views/settings/users/View.vue#L226

Reply:
Great catch, I didn't see it at first